features/fixtures/simple_app/spec/graphql_schema1_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Schema1 do
+  describe '.types' do
+    subject(:types) { described_class.types }
+
+    it 'includes Type1' do
+      expect(types).to include('Type1')
+    end
+
+    it 'executes queries' do
+      described_class.execute('{foo}')
```

Comment:
How does it work? Do you expect here something? Or you just run it to check no exception raised?